Affiliated-Project-Policy.md

```diff
@@ -0,0 +1,156 @@
+# OBF Affiliated Project Status Policy
```

Comment:
Guessing this will be removed once approval is in place?

Reply:
Yes.

Reply:
See 14eb733a9cb93d45b38887f1eabacffcad41f4cb.